xml/System.Security.Cryptography/HashAlgorithmName.xml

```diff
@@ -281,7 +281,7 @@ Due to collision problems with MD5 and SHA1, Microsoft recommends a security mod
       </ReturnValue>
       <Docs>
         <summary>Gets the underlying string representation of the algorithm name.</summary>
-        <value>The string representation of the algorithm name, or <see langword="null" /> or <see cref="F:System.String.Empty" /> if no hash algorithm is available.</value>
+        <value>The string representation of the algorithm name, or <see langword="default" /> if no hash algorithm is available.</value>
```

Comment:
@bartonjs I addressed your comment about null and empty, by removing the remark and (as suggested by @rpetrusha) modifying the value: substituted null and empty with default. Please confirm this was the correct change.

Reply:
I'd just leave it as it is entirely, or delete everything in the "or" clause.

Like all null values it's "do nothing", "do the default thing", or "be invalid", determined by the caller.